docs/api.md

```diff
@@ -52,10 +52,13 @@
     - [[zoom]](#zoom)
```

Comment:
Can you also update `CATALOG.md`, and replace `--user-agent <user agent of the latest Firefox>` with `--user-agent firefox` ?